docs/module/lag.md

```diff
@@ -69,10 +69,15 @@ links:
 ## Multi-chassis Link Aggregation (MLAG)
 
 For platforms that support it, link level redundancy can be provided by creating multiple links to different switches. The switches have an internal *peer link* to synchronize state related to the link aggregation, allowing them to present a single consistent network interface to the connected system.
+![image](https://github.com/user-attachments/assets/9c683f6e-0203-42e5-b5d6-91a2dfe711f8)
```

Comment:
There is no need for links to third-party websites. Add the image file to the 'module' directory and then link to it (see **netlab up** for an example).

Reply:
ok, added